app/controllers/bus_stops_controller.rb

```diff
@@ -56,15 +58,14 @@ def outdated
     respond_to do |format|
       format.html { render :outdated }
       format.csv do
-        send_data @stops.to_csv(limited_attributes: true),
-          filename: "outdated-stops-since-#{@date.strftime('%Y%m%d')}.csv"
+        send_data @stops.to_csv,
+                  filename: "outdated-stops-since-#{@date.strftime('%Y%m%d')}.csv"
       end
     end
   end
 
   def update
-    @stop.assign_attributes stop_params
-    if @stop.save
+    if @stop.update bsf_attrs
```

Comment:
Why is the `bsf_attrs` limitation necessary?

Reply:
wat?

Reply:
I think it was just a rename, @jforgue